daq/faucet_state_collector.py

```diff
@@ -71,10 +76,10 @@ def get_system(self):
     def get_topology(self):
         """get the topology state"""
         dplane_map = {}
-        dplane_map[TOPOLOGY_DP_MAP] = self.get_switch_map()
-        dplane_map[TOPOLOGY_LINK_MAP] = self.get_stack_topo()
-        dplane_map[TOPOLOGY_LACP] = None
-        dplane_map[TOPOLOGY_ROOT] = None
+        dplane_map[TOPOLOGY_DP_MAP] = self._get_switch_map()
+        dplane_map[TOPOLOGY_LINK_MAP] = self._get_stack_topo()
+        dplane_map[EGRESS_STATE], dplane_map[EGRESS_LAST_CHG], \
```

Comment:
line-break == bad, break out into a local variable or something?

Or, pass in the dplane_map into the function and call it self._update_egress_state(xxxx) -- and then modify it directly